README.md

```diff
@@ -1 +1,59 @@
-# School-Website
\ No newline at end of file
+## SCHOOL-WEBSITE
+
+![Elite school logo](img/logo.png)
+
+`Elite school` is the best school to nurture you child and enlight him/her academically.
+## Desktop Version
+![Desktop Version](img/desktop1.png)
+![Desktop Version](img/desktop2.png)
+![Desktop Version](img/desktop3.png)
+![Desktop Version](img/desktop4.png)
+## Mobile version
+![Desktop Version](img/mob1.png)
+![Desktop Version](img/mob2.png)
+![Desktop Version](img/mob3.png)
+![Desktop Version](img/mob4.png)
+## Usage
+
+---
```

Comment:
- [ ] Kindly add a live demo section containing deployed link of your portfolio, use GitHub Pages to deploy the web pages and all pages, links, and images work as expected.